packages/ui-components/package.json

```diff
@@ -9,7 +9,10 @@
   "repository": "https://github.com/paritytech/substrate-light-ui.git",
   "scripts": {
     "build": "tsc",
+    "build-storybook": "build-storybook -c .storybook -o .out",
     "start": "yarn build --watch",
+    "storybook": "start-storybook -p 9001 -c .storybook",
+    "deploy-storybook": "storybook-to-ghpages",
```

Comment:
It'd be nice to run this after each merge on master. What config do we need? Is a `GH_TOKEN` enough?

Reply:
It seems so AFAICT: https://docs.travis-ci.com/user/deployment/pages/

Reply:
Seems so, I added a commit with the updated travis config and added the token in the repo's travis settings